lib/Twig/Node/Expression/Call.php

```diff
@@ -10,18 +10,29 @@
  */
 abstract class Twig_Node_Expression_Call extends Twig_Node_Expression
 {
+    private $reflector;
```

Comment:
do we really need to make it stateful ? Attributes are mutable, but you don't reset this property when mutating the callable